e2e_deployment_files/stress_deployment.template.windows.json

```diff
@@ -0,0 +1,380 @@
+{
```

Comment:
If this file wasn't already here, I wouldn't add it. I'll bet it was removed from master for some reason... @and-rewsmith?

Reply:
Yes we should remove it.

Reply:
Removed.